lib/diagnostic.rb

```diff
@@ -95,8 +120,10 @@ def eat_meat(food)
 # Write your answer as a comment in the section below.
 
 ##
-# your answer, in comments, here
-#
+
+Advantages of using composition over direct inheritance : Modules are more managable. Inheritance
+lead to diamond shape inheritance. Modules are more flexible. Because include is a regular method
+call, Modules can be managed at runtime.
```

Comment:
Actually, Ruby doesn't have the 'diamond problem' because it only lets you inherit from one class. However, the rest of your answer is correct.